### ISSUE TITLE
Create comprehensive functionality test procedure

### ISSUE THREAD

**Issue by hajicj** [hajicj/MUSCIMarker#6]
Development documentation. Need to describe what kind of testing should be done before a feature is merged into master.

**Comment by hajicj** [comment]
Need to test actions:

1. App runs
2. Loads image
3. Zoom and move image
4. Annotate several objects with lasso
5. Annotate several objects with connected components
6. Select and unselect multiple symbols
7. Merge annotations destructively and non-destructively
8. Backup
9. Delete CropObject & recover from deletion.
10. Export CropObjects (to `test_data` dir)
11. Hide and show CropObjects
12. Clear CropObjects
13. Center image
14. Import CropObjects
15. Zoom back in
16. Open settings, change something and close settings
17. Exit
